changes/probe.py

```diff
@@ -58,11 +68,20 @@ def has_changelog():
 
 
 def has_readme():
-    """README.md"""
+    """README"""
+    readme = None
+    message = 'Create a README file'
+    for filename in os.listdir("."):
+        readme = re.match(r"README($|\.\w{2,})", filename, re.I)
```

Comment:
I had something similar in the beginning but thought a call to `os.path.exists` would amount to *iterate through os.listdir and check if any file matches* so I figured having just one call to `os.listdir` would cheaper. However I just looked at the Python source for `os.path.exists` [here](https://hg.python.org/cpython/file/v2.7.3/Lib/posixpath.py) and it actually just tries whether or not a `stat` fails for it in a try/except block so that might be more efficient. Your `any` is certainly more elegant, though. I'll do some timing, brb